__init__.py

```diff
@@ -259,6 +260,25 @@ class Fast64_ObjectProperties(bpy.types.PropertyGroup):
 
     sm64: bpy.props.PointerProperty(type=SM64_ObjectProperties, name="SM64 Object Properties")
     oot: bpy.props.PointerProperty(type=OOT_ObjectProperties, name="OOT Object Properties")
+    mk64: bpy.props.PointerProperty(type=MK64_ObjectProperties, name="MK64 Object Properties")
+
+
+class Fast64_MeshProperties(bpy.types.PropertyGroup):
+    """
+    Properties in object.fast64 (bpy.types.Mesh)
+    All new object properties should be children of this property group.
+    """
+
+    pass
```

Comment:
Will this be used later?

Reply:
Was planned to be used for collision or something. I am not mk64 minded.